frontend/packages/dev-console/integration-tests/reporter-config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "reporterEnabled": "mocha-junit-reporter, mochawesome",
+    "mochaJunitReporterReporterOptions": {
+      "mochaFile": "../../../gui_test_screenshots/junit_cypress-[hash].xml",
+      "toConsole": false
+    },
+    "mochawesomeReporterOptions": {
+      "reportDir": "../../../gui_test_screenshots/",
+      "reportFilename": "cypress_report",
```

Comment:
New issue discovered, please see [this](https://github.com/openshift/console/pull/6765#issuecomment-716553146)!

Please change `reportFilename` to "cypress_report_devconsole".  -thanks

Reply:
Sure, I will update it